src/routes/data-decode/entities/data-decoded.entity.ts

```diff
@@ -0,0 +1,12 @@
+import { ApiProperty } from '@nestjs/swagger';
+import {
+  DataDecoded as DomainDataDecoded,
+  DataDecodedParameter,
+} from '../../../domain/data-decoder/entities/data-decoded.entity';
+
+export class DataDecoded implements DomainDataDecoded {
+  @ApiProperty()
+  method: string;
+  @ApiProperty()
+  parameters?: DataDecodedParameter[] | undefined;
```

Comment:
```suggestion
  @ApiPropertyOptional()
  parameters?: DataDecodedParameter[];
```

Reply:
Changed in [1d66427](https://github.com/5afe/safe-client-gateway-nest/pull/132/commits/1d6642705e425416d8d1842dc5d02f6792202098)